client-data.md

```diff
@@ -1,21 +1,17 @@
 # Filecoin Client Data Processing
 
-Data that clients wish to store must be pre-processed before it can be sent to miners and have a storage deal made for it.
+Client must pre-process their data into a valid formats before sending them to miners, creating storage deals and generating mining proofs.
```

Comment:
This way this sentence is written sounds like a client is generating a mining proof, which I don't believe is true, is it ?

Reply:
yup, it's also redundant comment, removing it